src/App.js

```diff
@@ -4,19 +4,26 @@ import { Route, Routes } from 'react-router-dom';
 import Home from './Components/Home';
 import SingleCryptid from './Components/SingleCryptid';
 import AllSightings from './Components/AllSightings';
+import SingleSighting from './Components/SingleSighting'
 import NavBar from './Components/NavBar';
+import ErrorModal from './Components/ErrorModal'
 import { NavigationProvider } from './Context/NavigationContext';
+import { ErrorProvider } from './Context/ErrorContext'
 
 const App = () => {
   return (
-    <NavigationProvider>
-      <NavBar />
-      <Routes>
-        <Route path="/" element={<Home />} />
-        <Route path="/singlecryptid/:name" element={<SingleCryptid />} />
-        <Route path="/allsightings" element={<AllSightings />} />
-      </Routes>
-    </NavigationProvider>
+    <ErrorProvider>
```

Comment:
Ohhhh thanks for adding the error provider context api! Also the single sighting component!